### PR TITLE
working endpoint

### PR DESCRIPTION
**Pull request by allisondurham**
for testing this:

$>  python manage.py runserver
command click local server link

feel free to add test pieces in the admin

http://127.0.0.1:8000/api/places/         for all works

for any work in your database, visit /api/places/[id#]       for example:
http://127.0.0.1:8000/api/places/2/      for work #2, change integer for relevant piece

you are expecting to see a json response with all the fields and their corresponding data for either a list of all works, or a single specific one

### CODE REVIEW

File: durham/settings.py [hunk 1]
```diff
@@ -39,6 +39,9 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django_extensions',
+    'rest_framework',
+    # 'places.apps.PlacesConfig'
```
Comment (KimWheaton) [inline durham/settings.py]: You can remove this because you don't need both.

File: places/serializers.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from rest_framework import serializers
+from places.models import Work
+
+
+class WorkSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Work
+        fields = ['id', 'title', 'year', 'latitude', 'longitude', 'category', 'description', 'photo', 'notes', 'wheelchairAccessible']
+
+# class WorkSerializer(serializers.Serializer):
```
Comment (KimWheaton) [inline places/serializers.py]: You can remove this code.

**Review by KimWheaton — CHANGES_REQUESTED** [review]
Thanks, Allison. This looks really good. There are just two small changes. 
Also there needs to be an Artist class endpoint and Artist in the Work serializer.

File: durham/settings.py (hunk 1 above)
Comment (allisondurham) [reply durham/settings.py]: done, removed commented out code

File: places/serializers.py (hunk 2 above)
Comment (allisondurham) [reply places/serializers.py]: done!

**Comment by allisondurham** [comment]
I believe I addressed the requested changes

**Review by KimWheaton — APPROVED** [review]

**Comment by KimWheaton** [comment]
🦦

**Comment by KimWheaton** [comment]
🦦

**Pull request merged**